src/database/migrations/1.0.0/1658153656051-pluginsfirewalls.ts

```diff
@@ -0,0 +1,32 @@
+import {MigrationInterface, QueryRunner, TableColumn} from "typeorm";
+
+export class pluginsfirewalls1658153656051 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.addColumn('firewall', 
+            new TableColumn({
```

Comment:
@cmunyoz , @soltecsis-iortega what do you think about using a `boolean` column per plugin instead of this column?

Reply:
Given that we only need a flag per plugin, I think that is more optimal to use a flags unsigned integer like the one that we are already using for the options flags.